service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java

```diff
@@ -470,6 +472,22 @@ private boolean includeHearing(Element<HearingOrder> cmo, HearingBooking hearing
         return SEND_TO_JUDGE == cmo.getValue().getStatus() && cmo.getId().equals(hearing.getCaseManagementOrderId());
     }
 
+    private List<Element<SupportingEvidenceBundle>> buildSupportingEvidenceBundle(
+        List<Element<SupportingEvidenceBundle>> cmoSupportingDocs) {
+        String uploadedBy = documentUploadHelper.getUploadedDocumentUserDetails();
+
+        List<Element<SupportingEvidenceBundle>> supportingDocs = new ArrayList<>();
+        cmoSupportingDocs.forEach(cmoSupportingDoc -> supportingDocs.add(
+            element(cmoSupportingDoc.getId(), SupportingEvidenceBundle.builder()
```

Comment:
probably better to have
moSupportingDoc.getValue().toBuilder().dateTimeUploaded().uploadedBy().build()

otherwise you can lost some properties if new added to class